src/state_machine.zig

```diff
@@ -459,7 +460,7 @@ pub const StateMachine = struct {
         return self.accounts.getPtr(id);
     }
 
-    /// See the comment for get_account().
+    /// See the comment for get_transfer().
```

Comment:
We are actually intending to reference the comment for `get_account()` here.

Reply:
Fixed.